webpack.config.js

```diff
@@ -20,7 +20,6 @@ module.exports = {
   module: {
     loaders: [
       {
-        exclude: /node_modules/,
```

Comment:
The `externals` option excludes node_modules from build anyway.

But anyway you should not use `exclude` but `include` instead.